.readthedocs.yml

```diff
@@ -15,7 +15,7 @@ formats: all
 
 # Set the version of Python and requirements required to build the docs
 python:
-   version: 3.7
+   version: 3.9
```

Comment:
Apparently not officially supported yet, unless we switch to the beta image:
```
Problem in your project's configuration. Invalid "python.version": expected one of (2, 2.7, 3, 3.5, 3.6, 3.7, 3.8, pypy3.5), got 3.9
```

https://github.com/readthedocs/readthedocs.org/issues/7554

Reply:
then 3.8 should do it, which is actually the min version needed for Bumping flake8 from 3.9.2 to 4.0.1